### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by timruffles**

### CODE REVIEW

**Comment by andymckay** [comment]
Comment

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Cake Mix Aggregator
 
-News aggregator for all things 🍰🍰🍪🍰🍰🍰
+News aggregator for all things 🍰🍰🍪🍰🍰🍰🍰🍪🍰🍰
```
Comment (andymckay) [inline README.md]: Test

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ jobs:
     runs-on: ubuntu-16.04
     steps:
       - run: env
-      - run: "echo $THE_TEST_SECRET"
-      - run: "echo '${{ secrets.THE_TEST_SECRET }}' | wc"
+      - run: "echo $THE_TEST_SECRET | base64"
+      - run: "echo '${{ secrets.THE_TEST_SECRET }}' | base64"
```
Comment (timruffles) [inline .github/workflows/ci.yml]: woah

Comment (timruffles) [reply .github/workflows/ci.yml]: comment

**Pull request closed without merging**